mira/metamodel/ops.py

```diff
@@ -560,8 +565,12 @@ def counts_to_dimensionless(tm: TemplateModel,
                     # for the concept and if so, we normalize it as well
                     if concept.name in tm.initials and concept.name not in initials_normalized:
                         init = tm.initials[concept.name]
-                        if init.value is not None:
-                            init.value /= (norm_factor ** exponent)
+                        if init.expression is not None:
+                            try:
```

Comment:
I don't think I understand this part, why can't the operations simply be applied to the sympy expression? I don't think we need to turn it into a float.